Duplicate key in docker-compose.yml

<pre>
⋊> git log -1 --oneline
047e7f7 Merge pull request #489 from laradock/yarn

⋊> pip install yamllint

⋊> yamllint docker-compose.yml
docker-compose.yml
  1:1       warning  missing document start "---"  (document-start)
  5:1       warning  comment not indented like content  (comments-indentation)
  11:1      warning  comment not indented like content  (comments-indentation)
  <b>36:81     error    line too long (90 > 80 characters)  (line-length)</b>
  <b>39:12     error    wrong indentation: expected 12 but found 11  (indentation)</b>
  42:1      warning  comment not indented like content  (comments-indentation)
  <b>64:81     error    line too long (90 > 80 characters)  (line-length)</b>
  <b>67:81     error    line too long (84 > 80 characters)  (line-length)</b>
  70:1      warning  comment not indented like content  (comments-indentation)
  88:1      warning  comment not indented like content  (comments-indentation)
  105:1     warning  comment not indented like content  (comments-indentation)
  116:1     warning  comment not indented like content  (comments-indentation)
  <b>119:15    error    trailing spaces  (trailing-spaces)</b>
  131:1     warning  comment not indented like content  (comments-indentation)
  145:1     warning  comment not indented like content  (comments-indentation)
  158:1     warning  comment not indented like content  (comments-indentation)
  171:1     warning  comment not indented like content  (comments-indentation)
  183:1     warning  comment not indented like content  (comments-indentation)
  192:1     warning  comment not indented like content  (comments-indentation)
  201:1     warning  comment not indented like content  (comments-indentation)
  216:1     warning  comment not indented like content  (comments-indentation)
  227:1     warning  comment not indented like content  (comments-indentation)
  237:1     warning  comment not indented like content  (comments-indentation)
  252:1     warning  comment not indented like content  (comments-indentation)
  261:1     warning  comment not indented like content  (comments-indentation)
  <b>273:9     error    duplication of key "volumes" in mapping  (key-duplicates)</b> <---
  279:1     warning  comment not indented like content  (comments-indentation)
  296:1     warning  comment not indented like content  (comments-indentation)
  305:1     warning  comment not indented like content  (comments-indentation)
</pre>

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -261,14 +261,12 @@ services:
 
     caddy:
         build: ./caddy
-        volumes_from:
-            - applications
-        volumes:
-            - caddy:/root/.caddy
         ports:
             - "80:80"
             - "443:443"
             - "2015:2015"
+        volumes_from:
+            - applications
         volumes:
             - ./caddy/Caddyfile:/etc/Caddyfile
             - ./logs/caddy:/var/log/caddy

